server.py

```diff
@@ -413,46 +413,46 @@ def callback(self):
-                                if verbose:     
-                                    print "[Callback] ChID:", self.qItem[3 * i], "UnitID:", self.qItem[3 * i + 1], "Time:", self.qItem[3 * i + 2] * 1000, "t: ", h.t         
+                                    s.inncl[localIndex].event(timeStamp, 1)
+                                if verbose:
+                                    print("[Callback] ChID:", self.qItem[3 * i], "UnitID:", self.qItem[3 * i + 1], "Time:", self.qItem[3 * i + 2] * 1000, "t: ", h.t)
                                     #fdtime.write(str(int(self.qItem[3*i])) + '\t' +str(int(self.qItem[3*i+1])) + '\t' +str('{:g}'.format(self.qItem[3*i+2])) + '\n')
                             else: # spike ignored. For dubug
                                 if verbose:
-                                    print "[Callback] ChID:", self.qItem[3 * i], "UnitID:", self.qItem[3 * i + 1], "Time:", self.qItem[3 * i + 2], "t: ", h.t
+                                    print("[Callback] ChID:", self.qItem[3 * i], "UnitID:", self.qItem[3 * i + 1], "Time:", self.qItem[3 * i + 2], "t: ", h.t)
                                     #fdtime.write(str(int(self.qItem[3*i])) + '\t' +str(int(self.qItem[3*i+1])) + '\t' +str('{:g}'.format(self.qItem[3*i+2])) + '\t' + str('{:g}'.format(h.t/1000)) + '\n')
                     # move t to currSimTime
                     if currSimTime > h.t:# and currSimTime < duration:
-                        if simMode == 1: # online mode                           
+                        if simMode == 1: # online mode
                             newCurrTime.value = currSimTime
                     else:
-                        currSimTime = h.t  
+                        currSimTime = h.t
                         newCurrTime.value = currSimTime
                 finally:
                     with lock:
                         currNeuronTime.value = currSimTime # (ms)
                         if verbose:
-                            print "[Callback] currNeuronTime: ", currSimTime, nextInvl
+                            print("[Callback] currNeuronTime: ", currSimTime, nextInvl)
         except:
             if isCommunication == 0:
                 self.fd.close()
             if verbose:
-                print "[Callback] exception occurs:", traceback.print_exc()
+                print("[Callback] exception occurs:", traceback.print_exc())
         finally: # update current neuron time
             if isCommunication == 0:
                 pass
             else:
                 if newCurrTime.value + nextInvl <= duration:
-                    h.cvode.event(newCurrTime.value + nextInvl, self.callback)    
+                    h.cvode.event(newCurrTime.value + nextInvl, self.callback)
                 if verbose:
-                    print('[callback.h.t End(id= %i)] %.3f' %(pc.id(), h.t))                   
+                    print(('[callback.h.t End(id= %i)] %.3f' %(pc.id(), h.t)))
```

Comment:
pedantic point, but this change isn't necessary; it was already in parentheses

Reply:
I think it's a side effect from the `2to3` utility.